tests/regression_tests/plot_voxel/test.py

```diff
@@ -0,0 +1,65 @@
+import glob
+import hashlib
+import os
+from subprocess import check_call
+import importlib
+
+import h5py
+import openmc
+import pytest
+
+from tests.testing_harness import TestHarness
+from tests.regression_tests import config
+
+
+pytestmark = pytest.mark.skipif(
+    importlib.util.find_spec('vtk') is None,
+    reason="vtk module is not installed.")
```

Comment:
pytest makes this slightly simpler: see [here](https://docs.pytest.org/en/documentation-restructure/how-to/skipping.html#skipping-on-a-missing-import-dependency)